app/views/home/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Welcome to EventGo</h1>
```

Comment:
if possible, try to implement i18n after discussing with your mentor bhai

Reply:
Ok bhai.